package.json

```diff
@@ -103,6 +110,7 @@
   "dependencies": {
     "deepmerge": "^2.0.1",
     "load-script": "^1.0.0",
+    "maestro-videojs-vast": "0.0.6-alpha",
```

Comment:
can I see this guy?

Reply:
https://github.com/lessthan3/maestro-videojs-vast